### ISSUE TITLE
0.10.1 Does Not Honor translatedPathPrefix

### ISSUE THREAD

**Issue by commonsguy** [doppllib/doppl-gradle#16]
The [adding Doppl to an existing project](http://doppl.co/docs/AddingDoppl) instructions use this `dopplConfig`:

```groovy
dopplConfig {
  translatePattern {
    include 'co/doppl/so/api/**'
    include 'co/doppl/so/arch/**'
    include 'co/doppl/so/RepositoryTest.java'
  }

  translatedPathPrefix 'co.doppl.so.api', 'SOAPI'
  translatedPathPrefix 'co.doppl.so.arch', 'SOA'

  testIdentifier {
    include 'co/doppl/so/RepositoryTest.java'
  }
}
```

If you use 0.9.3 of the plugin, and you look at the code-generated `CoDopplSoRepositoryTest.m` file in the `j2objcSrcGenTest` directory, you will see that it has `#include "SOAQuestion.h"` and `#include "SOARepository.h"`, as expected.

If you use 0.10.1 of the plugin, and you look at that same file, you see see `#include "CoDopplSoArchQuestion.h"` and `#include "CoDopplSoArchRepository.h"` &mdash; basically, what would be there if there were no `translatedPathPrefix` statements. However, those statements exist, and the actual generated file has the proper names (e.g., `SOAQuestion.h`), so Xcode cannot resolve the `#import` lines.

See issue #15 for 0.9.3 output, and see the attached `SOAndroid-1.0.1.zip` for the 0.10.1 output.

[SOAndroid-1.0.1.zip](https://github.com/doppllib/doppl-gradle/files/1515068/SOAndroid-1.0.1.zip)

**Comment by commonsguy** [comment]
This is still an outstanding issue with 0.10.3.

**Comment by commonsguy** [comment]
Fixed in 0.10.3.
